src/cloud-code/Parse.Cloud.js

```diff
@@ -8,40 +8,205 @@ function getClassName(parseClass) {
   return parseClass;
 }
 
+/** @namespace
+ * @name Parse
+ * @description The Parse SDK.
+ *  see [api docs](https://docs.parseplatform.org/js/api) and [guide](https://docs.parseplatform.org/js/guide)
+ */
+
+/** @namespace
+ * @name Parse.Cloud
+ * @memberof Parse
+ * @description The Parse Cloud Code SDK.
+ */
+
 var ParseCloud = {};
+/**
+ * Defines a Cloud Function.
+ *
+ * **Available in Cloud Code only.**
+ * **Starting parse-server 3.0.0, all cloud hooks and functions are asynchronous**
+ * **All response objects are removed and replaced by promises / async/await based resolution**
```

Comment:
All response objects are removed and replaced by promises and optionally async/await based resolution.